lint.js

```diff
@@ -81,7 +82,11 @@ const command = async (file, cmd) => {
 
     linter.initialize();
 
-    await loader.loadRuleFiles(cmd.rules, { verbose });
+    if (config.fileExists()) {
```

Comment:
I think this might be exposing too much internal logic to the end user. 

``` javascript
const rules = [].concat(config.getConfigRules(), cmd.rules);
await loader.loadRuleFiles(cmd.rules, { verbose });
```

Then we don't need to worry about if the file exists, it'd just be empty if theres no file, or if the file contains no rules.

Reply:
I see your point. Since `rules` is a new array, shouldn't it be:
```
const rules = [].concat(config.getConfigRules(), cmd.rules);
await loader.loadRuleFiles(rules, { verbose });
```